GoogleForCreators/web-stories-wp
# WIP: Redesigned grid view

## Context

This implements the re-designed grid view mode.

*Note 1*: Still WIP.

*Note 2*: This includes the slider PR from #6442, but will be rebased from the main branch once that PR is merged.

## To-do

See todo in #6206.

## User-facing changes

_Screenshots incoming_

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Add multiple pages to a story (and make sure they're distinguishable)
2. Open the grid view
3. Interact and observe it working as per spec

### UAT






## Checklist




---

Fixes #6206
